src/main/scala/com/amazon/deequ/suggestions/ConstraintSuggestionRunBuilder.scala

```diff
@@ -45,6 +45,8 @@ class ConstraintSuggestionRunBuilder(val data: DataFrame) {
   protected var saveConstraintSuggestionsJsonPath: Option[String] = None
   protected var saveEvaluationResultsJsonPath: Option[String] = None
   protected var kllParameters: Option[KLLParameters] = None
+  protected var predefinedColumnDataTypes: Map[String, DataTypeInstances.Value] =
```

Comment:
please rename to predefinedTypes and change default argument to Map.empty

Reply:
Sure.